libs/app-shell/src/lib/api/extensions/lookup/plugin-descriptor.model.ts

```diff
@@ -1,31 +1,50 @@
-export interface PluginDescriptor {
+export interface AngularIvyComponentDescriptor {
+    /** Uniq name of module in plugin */
     name: string;
-    provider: string
-    version: string;
-    sinceVersion: string;
-    untilVersion: string;
-    remoteEntry: string;
-    remoteName: string;
-    remoteRoute: string;
+    /** Type of a plugin module */
+    type: 'angular-ivy-component';
+    /** Exposed Angular Module */
     exposedModule: string;
-    displayName: string;
+    /** Component name we are planning to inject */
     componentName: string;
-
-    type: Scope;
-    category: string;
-    hasRoutes: boolean;
-    changeNotes: string;
+    /** most likely this property will be removed, but as for this this is a way to register route for a component*/
+    route?: string;
```

Comment:
I think you mixed hasRoute original property, but this Route is still needed , this is our current routing solution

Reply:
Yes it's, but it's rather optional. Right?

Reply:
yes tru tru